### PR TITLE
Alphabetizes Plugins.rst

### PR DESCRIPTION
**Pull request by cppcooper**
I alphabetized things in distinct clear cut commits. First each section [separately] as they were in the file, then finally the order of sections.

edit: sorry they don't match +2,212 −2,218 it's hopefully whitespace O_O

### CODE REVIEW

File: docs/Plugins.rst [hunk 1]
```diff
@@ -302,6 +291,18 @@ Simple plugin to automate following random dwarves. Most of the time things will
...
+stonesense
+==========
+An isometric visualizer that runs in a second window. Usage:
+
+:stonesense:        Open the visualiser in a new window.  Alias ``ssense``.
+:ssense overlay:    Overlay DF window, replacing the map area.
+
+For more information, see `the full Stonesense README <stonesense>`.
+
+
```
Comment (cppcooper) [inline docs/Plugins.rst]: extra space

File: docs/Plugins.rst [hunk 2]
```diff
@@ -451,14 +458,6 @@ Subcommands that persist until disabled or DF quits:
...
 .. comment: sort these alphabetically
 
-.. _fix-armory:
-
-fix-armory
-==========
-`This plugin requires a binpatch <binpatches/needs-patch>`, which has not
-been available since DF 0.34.11
-
-
```
Comment (cppcooper) [inline docs/Plugins.rst:460]: extra space

File: docs/Plugins.rst [hunk 3]
```diff
@@ -473,6 +473,42 @@ UI Upgrades
    :local:
```
Comment (cppcooper) [inline docs/Plugins.rst]: commit: ef68785b30d554754de3b9481cd65065b269b0ed
no extra lines added/removed

File: docs/Plugins.rst [hunk 4]
```diff
@@ -1874,226 +2092,6 @@ Examples
...
-* ``autofarm runonce``:
-    Updates all farm plots once, without enabling the plugin
-* ``autofarm status``:
-    Prints status information, including any applied limits
-* ``autofarm default 30``:
-    Sets the default threshold
-* ``autofarm threshold 150 helmet_plump tail_pig``:
-    Sets thresholds of individual plants
-
-
```
Comment (cppcooper) [inline docs/Plugins.rst:2096]: commit: ec80cc5fc247b37c185329227d8388604e32ace2
2 extra spaces

File: docs/Plugins.rst [hunk 5]
```diff
@@ -2248,35 +2276,6 @@ Example:
...
-:quality, q:   Change base quality. Must be followed by number (0-5).
-:force:        Ignore subtypes, force change to new material.
-
-Examples:
-
-``changeitem m INORGANIC:GRANITE here``
-   Change material of all items under the cursor to granite.
-``changeitem q 5``
-   Change currently selected item to masterpiece quality.
-
```
Comment (cppcooper) [inline docs/Plugins.rst:2281]: commit: ab65f7fe071d8930b3a35e7231c06906e202b73d
extra space removed

File: docs/Plugins.rst [hunk 6]
```diff
@@ -2323,40 +2322,6 @@ Basic commands:
...
-Examples:
-
-``expdig diag5 hidden``
-  Designate the diagonal 5 patter over all hidden tiles
-``expdig``
-  Apply last used pattern and filter
-``expdig ladder designated``
-  Take current designations and replace them with the ladder pattern
-
```
Comment (cppcooper) [inline docs/Plugins.rst:2361]: commit: ab65f7fe071d8930b3a35e7231c06906e202b73d
extra space removed

File: docs/Plugins.rst [hunk 7]
```diff
@@ -2438,6 +2415,28 @@ Usage:
...
+Options:
+
+:dig:
+:channel:
+:ramp:
+:updown: up/down stairs
+:up:     up stairs
+:down:   down stairs
+:clear:  clear designation
+
```
Comment (cppcooper) [inline docs/Plugins.rst:2425]: commit: ab65f7fe071d8930b3a35e7231c06906e202b73d
extra space added

net: -1

**Comment by lethosor** [comment]
I confirmed locally that the changes are whitespace-only with:
```
diff -u <(git show origin/develop:docs/Plugins.rst | sort) <(git show cppcooper/alphabetize-docs:docs/Plugins.rst | sort)
```
which prints
```
@@ -799,12 +799,6 @@
 
 
 
-
-
-
-
-
-
             {
             }
         ]
```

so yes, 6 blank lines lost. Shouldn't be an issue as long as the affected sections continue to render the same way as before.

**Comment by cppcooper** [comment]
> I confirmed locally that the changes are whitespace-only with:
> 
> ```
> diff -u <(git show origin/develop:docs/Plugins.rst | sort) <(git show cppcooper/alphabetize-docs:docs/Plugins.rst | sort)
> ```

Oh my, black magic like this would have been helpful before I started manually checking the diffs. I was down to just one missing deletion

**Comment by cppcooper** [comment]
> affected sections continue to render the same way as before

I just installed sphinx and enabled building docs. Things seems to be rendering just fine, all I did was look over the whole file though. I'm not quite sure which sections are affected specifically.

**Comment by cppcooper** [comment]
So I've gone through and verified that all entries are present and accounted for. Since the lines are at the boundaries between these entries, they would not exist in the table of contents if there were a problem from the removed lines.

**Review by myk002 — APPROVED** [review]

**Pull request merged**